script.js

```diff
@@ -161,5 +188,6 @@ const dd = String(currentDate.getDate()).padStart(2, '0');
 const mm = String(currentDate.getMonth() + 1).padStart(2, '0');
 const yyyy = currentDate.getFullYear();
 
-today = `${yyyy}-${mm}-${dd}`;
-document.getElementById('startedOn').value = today;
+today = `${yyyy}-${mm}-${dd}`; 
```

Comment:
Have we declared `today` as `const` or `let` somewhere?

Reply:
no

Reply:
Let us do that please :)
Let's initialize variables and use

Reply:
okay

Reply:
yes, i have made the changes.